symphony-bdk-core/pom.xml

```diff
@@ -97,6 +97,12 @@
             <version>${jackson.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
I don't think it will bring a lot of dependencies with it. I used only artifact `resilience4j-retry` and from what I see in [resilience4j-retry](https://mvnrepository.com/artifact/io.github.resilience4j/resilience4j-retry/1.5.0), it doesn't include many dependencies